tests/bearlib/abstractions/external_bear_wrap_testfiles/ExternalBearWrapTest.py

```diff
@@ -0,0 +1,105 @@
+import os
+import json
+import unittest
+
+from coalib.bearlib.abstractions.ExternalBearWrap import external_bear_wrap
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+
+
+def get_testfile_name(name):
+    """
+    Gets the full path to a testfile inside the same directory.
+
+    :param name: The filename of the testfile to get the full path for.
+    :return:     The full path to given testfile name.
+    """
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                        name)
+
+
+class ExternalBearWrapComponentTest(unittest.TestCase):
+
+    class Dummy:
+        pass
+
+    class TestBear:
```

Comment:
Yes, but I refactored the code a bit. Now the class will be responsible for overwriting `create_arguments()` similar to linter. I found this to be a better way since it makes it easier to use languages that use interpreters.